status/status.go

```diff
@@ -216,3 +220,17 @@ func GenerateSummary(state intgteststat.IntegrationTestStatus, snapshotName, sce
 
 	return summary, nil
 }
+
+// GenerateURLrefferenceToConsole returns string link to integration test pipelinerun logs for tenant
+func GenerateURLrefferenceToConsole(application string, pipelinerun string, namespace string) string {
```

Comment:
The content of this function is similar to `FormatPipelineURL()`, should we remove one of them?

Reply:
Thats exactly what I did. -- GenerateURLrefferenceToConsole